recipes/julia/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+export C_INCLUDE_PATH=${PREFIX}/include
+export LD_LIBRARY_PATH=${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
+export CMAKE_PREFIX_PATH=${PREFIX}
+
+make -j 4 prefix=${PREFIX} MARCH=core2 sysconfigdir=${PREFIX}/etc NO_GIT=1 \
+ USE_SYSTEM_LIBGIT2=1 USE_LLVM_SHLIB=0 USE_SYSTEM_CURL=1 USE_SYSTEM_MPFR=0 \
+ USE_SYSTEM_PATCHELF=1 USE_SYSTEM_LIBSSH2=1 USE_SYSTEM_LLVM=0 USE_SYSTEM_BLAS=1 \
```

Comment:
That would be my guess too. We may want to add `patchelf` as a `build` requirement then.